### PR TITLE
[bn]: Translation updated

### PR DESCRIPTION
**Pull request by rHafijur**
Some extra spaces removed.
Some translations improved / fixed.

### CODE REVIEW

File: locales/bn/json.json [hunk 1]
```diff
@@ -212,15 +212,15 @@
     "Edit Attached": "সংযুক্ত সম্পাদনা করুন",
     "Edit Profile": "প্রোফাইল সম্পাদনা",
     "Editor": "সম্পাদক",
-    "Editor users have the ability to read, create, and update.": "সম্পাদক ব্যবহারকারীদের পড়তে তৈরি, এবং আপডেট করার ক্ষমতা রাখে ।",
+    "Editor users have the ability to read, create, and update.": "সম্পাদক ব্যবহারকারীরা পড়ার, তৈরির, এবং আপডেট করার ক্ষমতা রাখে।",
```
Comment (arman-arif) [inline locales/bn/json.json:215]: "Editor users have the ability to read, create, and update.": "সম্পাদক ব্যবহারকারীরা পড়ার, তৈরির, এবং আপডেট করার ক্ষমতা রাখে।",
Here "আপডেট" can be replaced with "সম্পাদনা"

File: locales/bn/json.json [hunk 2]
```diff
@@ -212,15 +212,15 @@
...
     "Egypt": "মিশর",
     "El Salvador": "এল সালভাদর",
     "Email": "ই-মেইল",
     "Email Address": "ই-মেইল ঠিকানা",
     "Email Addresses": "ই-মেইল ঠিকানা সমূহ",
     "Email Password Reset Link": "ইমেল পাসওয়ার্ড রিসেট লিংক",
     "Enable": "সক্রিয় করুন",
-    "Ensure your account is using a long, random password to stay secure.": "আপনার অ্যাকাউন্ট নিরাপদ থাকার একটি দীর্ঘ, এলোমেলো পাসওয়ার্ড ব্যবহার করা হয় তা নিশ্চিত করুন ।",
+    "Ensure your account is using a long, random password to stay secure.": "নিরাপদ থাকতে, আপনার অ্যাাকাউন্টে দীর্ঘ, এলোমেলো পাসওয়ার্ডের ব্যাবহার নিশ্চিত করুন।",
```
Comment (arman-arif) [inline locales/bn/json.json:223]: Here please change the word "থাকতে" to "রাখতে"

File: locales/bn/json.json [hunk 3]
```diff
@@ -239,15 +239,15 @@
     "Filename": "ফাইলের নাম",
     "Finish enabling two factor authentication.": "দুটি ফ্যাক্টর প্রমাণীকরণ সক্ষম করা শেষ করুন।",
     "Finland": "ফিনল্যান্ড",
-    "For your security, please confirm your password to continue.": "আপনার নিরাপত্তার জন্য, অনুগ্রহ করে আপনার পাসওয়ার্ড চালিয়ে যান ৷",
+    "For your security, please confirm your password to continue.": "আপনার নিরাপত্তার জন্য, অনুগ্রহ করে আপনার পাসওয়ার্ড কনফার্ম করে চালিয়ে যান৷",
```
Comment (arman-arif) [inline locales/bn/json.json:242]: @rHafijur please change "কনফার্ম করে" to "নিশ্চিত করে"

**Review by arman-arif — CHANGES_REQUESTED** [review]
@rHafijur please change the words I mentioned above. If you have any other thought let me know.

File: locales/bn/json.json (hunk 1 above)
Comment (rHafijur) [reply locales/bn/json.json:215]: should i make another pull request for this?

File: locales/bn/json.json (hunk 2 above)
Comment (rHafijur) [reply locales/bn/json.json]: done

**Review by arman-arif — APPROVED** [review]
@andrey-helldar, you can merge now.

**Review by andrey-helldar — APPROVED** [review]
তোমাকে ধন্যবাদ

**Pull request merged**